css/styles.css

```diff
@@ -58,15 +60,18 @@
     margin: 0;
     font-size: 14px;
     font-weight: normal;
+    line-height: 20px;
     text-transform: uppercase;
 }
 .ab-column h3.icon:before {
-    font: normal 28px/28px icons;
+    font: normal 30px/30px icons;
     top: 4px;
-    right: 0;
+    right: 20px;
     speak: none;
+    width: 1em;
```

Comment:
Removing this 1em in width results in slightly shifting icons. Since it's unknown if this causes problems with some browser/OS combinations, I'd opt for leaving this in our CSS for the moment. (This again was part of the CSS until 2d193ccc36be60025bd5d8d82f6238b0203b5ffb)